### ISSUE TITLE
Building and verifying "Pashto"

### ISSUE THREAD

**Issue by burrage** [curiouslearning/FeedTheMonsterJS#749]
See the latest notes below.

**Acceptance Criteria** (for launch to Prod)
Given that the production server now has the files that were on development for Pashto,
When I add the prod URL to the prod manifest and I select "Pashto" as a language,
Then I should see the app icon for FTM.

Given that I see the app icon for FTM,
When I open the app and play through the first level in its entirety,
Then I should see text in Pashto and hear Pashto audio.

**Comment by burrage** [comment]
Currently waiting on Content revisions.

**Comment by burrage** [comment]
We are missing one piece of audio...

**Comment by burrage** [comment]
Pashto QA came back, we had very minor changes in the levelgen, and they're already done! Just need to re-run the content_verification process and redeploy!

**Comment by burrage** [comment]
If we rebuild the JSON and @Ashish1998 can verify we do not the issues listed at the top of the QA (levelgen change and different audio files), we CAN go ahead and push this to prod WITHOUT me OK'ing it.

**Comment by burrage** [comment]
@Nikhil77sutara Hey Nikhil, just wanted to check on something... did the latest audio get uploaded for prod and the new JSON generated before we pushed to prod (see last two comments)? Did someone QA to make sure the new audio was in? 

I checked the development version and I'm still hearing old audio so I'm concerned there may have been a misunderstanding and the last few notes were skipped before pushing to prod.

Thanks Nikhil!

**Comment by Nikhil77sutara** [comment]
> @Nikhil77sutara Hey Nikhil, just wanted to check on something... did the latest audio get uploaded for prod and the new JSON generated before we pushed to prod (see last two comments)? Did someone QA to make sure the new audio was in?
> 
> I checked the development version and I'm still hearing old audio so I'm concerned there may have been a misunderstanding and the last few notes were skipped before pushing to prod.
> 
> Thanks Nikhil!

Hey 
[@bburrage](https://chimple.slack.com/team/U3MH133ME)
 ,
We have checked the audio’s in the Pashto language on the drive in the new letters sounds folder and find out that the voice actor is same on production as in the drive

**Comment by Nikhil77sutara** [comment]
Hey @burrage,
We have checked the audio’s in the Pashto language on the drive in the new letters sounds folder and find out that the voice actor is same on production as in the drive

**Comment by burrage** [comment]
Hey @Nikhil77sutara, I'm seeing two potential problems here... let me know if you are seeing this similarly:

1.) I can tell the wrong audio is on the production server. Using the QA report as a guide as was asked above...

I'm listening to the audio file for the character ک  which the QA report says, "The word ک should sound like 'Ka,' not 'yaiy'"

Here's the audio that's in the /lang/pashto/audios/ folder in the production FTM bucket:

https://s3.console.aws.amazon.com/s3/object/feedthemonster-production?region=ap-south-1&prefix=lang/pashto/audios/%DA%A9.mp3

This audio still sounds like 'yaiy' does it not? 

Am I looking in the wrong bucket or is it correct that this is the wrong audio?

2.) There is also a potential content organization problem. I'm looking in the Pashto audio folder on Google Drive and I'm noticing there are multiple folders that have a character named ک . Some are even with a different voice actor. This seems problematic for our content checking scripts because how would it know which character to choose?

Is this also correct that this is a problem that we need to address on the content organization side? If so, how do you purpose we solve it given how the content checking script operates? CC: @amitsinghsutara for awareness.

**Comment by Nikhil77sutara** [comment]
Hey @burrage ,
You are correct the audio is mismatching, unfortunately  we were not able to figure it  out earlier  before launching to production,
Also Ben you were correct about multiple folders having same file name with different voice actor's!
In order to resolve this issue will it be possible for you to suggest us the correct audio folder where we can find the correct updated audio assets!

**Comment by burrage** [comment]
I will spin up a new task card when Stephanie has compiled all the correct audio in a single location in the new year.
